kubernetes/helm/values.override_production.auction-ui.k8s.sidestream.tech.yaml

```diff
@@ -0,0 +1,22 @@
+# Value overrides for the deployment to the following host:
+hosts:
+  - "production.auction-ui.k8s.sidestream.tech"
+  - "auctions.makerdao.network"
```

Comment:
@valiafetisov note how now two hosts are defined here.

I did a test deployment via the CD pipeline in the previous commits. To test it out:
```sh
> curl 35.156.101.56 -H 'Host: auctions.makerdao.network' -v
> curl 35.156.101.56 -H 'Host: production.auction-ui.k8s.sidestream.tech' -v 
```

Both should end up redirecting you within the cluster (due to tls). If you add `-L` you will also end up visiting the page.

Reply:
But that is only about traffic on port 80 🙁

Reply:
In a short term I could not come up with a better solution to proof to you that we're listening to both hosts, as we do not have control over their domain and TLS certs (see below) 😢